fdbserver/workloads/BlobRestoreWorkload.actor.cpp

```diff
@@ -106,40 +107,92 @@ struct BlobRestoreWorkload : TestWorkload {
 					return Void();
 				}
 			}
-			wait(delay(1));
+			wait(delay(5));
```

Comment:
Why arbitrary delay? It'd be good to at least have a comment explaining why this is necessary at least, but making it so we were actually waiting on whatever the desired end state condition is instead of just sleeping would be better.